src/shared/port-finder.js

```diff
@@ -1,8 +1,11 @@
 import { ListenerCollection } from './listener-collection';
 import { isMessageEqual } from './port-util';
 
-const MAX_WAIT_FOR_PORT = 1000 * 10;
-const POLLING_INTERVAL_FOR_PORT = 250;
+/** Timeout for waiting for the host frame to respond to a port request. */
+export const MAX_WAIT_FOR_PORT = 1000 * 20;
```

Comment:
I will go for even longer, 30 seconds.

On my machine loading on an iframe https://html.spec.whatwg.org/ can take around 10 seconds, if you add a slow network even minutes.